src/main/scala/firrtl/passes/PadWidths.scala

```diff
@@ -9,7 +9,6 @@ import firrtl.Mappers._
 
 // Makes all implicit width extensions and truncations explicit
 object PadWidths extends Pass {
-  def name = "Pad Widths"
```

Comment:
Did you want to delete all of these names? You could change them all to override if you liked the names we gave.